.editorconfig

```diff
@@ -18,6 +18,5 @@ ij_continuation_indent_size = 4
 ij_continuation_indent_size = 4
 ij_kotlin_name_count_to_use_star_import = 99
 ij_kotlin_name_count_to_use_star_import_for_members = 99
-# These are currently disabled ktlint rules, used for: git precommit hook (see `tools/ktlint-precommit-hook.sh`),
```

Comment:
I agree the comment here is probably not a good place, but maybe this could be moved to some more relevant location, maybe some dev guidelines?

Reply:
I'm not sure about this one. Like I mentioned on the other PR, we don't want to ask people to do more than checkout to be able to work on our project. Also, ktlintCheck runs as part of the normal Gradle build, and people should build locally before pushing. If they don't, that'll fail in PR builds anyway.

So, for now, I'll keep the hook and the comment as hints for those who want, and we'll see how that works on master. I'm happy to change it later if linting becomes more of a pain.